### ISSUE TITLE
[Bug Report]: 

### ISSUE THREAD

**Issue by JeffreyPang88** [ndeadly/MissionControl#408]
### Switch Firmware Version

13.2.0 (Latest)

### Atmosphère Version

1.2.5 (Latest)

### Mission Control Version

0.6.3 (Latest)

### Boot Method

Fusée

### Issue Description

Hi hi..I just have question about the pairing moga hero controller using missioncontrol..not sure is setting or other reason that cause it...the pairing is success, got the pairing success show on the screen.but after 2 seconds it disconnect itself..i try reinstall the missioncontrol and I try charge full battery for the moga but still no luck..😅

**Comment by ndeadly** [comment]
Sounds like another controller that doesn't like the `SetTsi` command. See the comment here https://github.com/ndeadly/MissionControl/issues/401#issuecomment-984755724 for an explanation and potential temporary fix. If it works let me know and I'll fix it in code.

**Comment by JeffreyPang88** [comment]
Wow, its works, thx a lot..appreciate for yrs help..😊..after I created a file in the directory folder.the moga controller can now connected without issue..thx..

**Comment by JeffreyPang88** [comment]
btw, the ps3 dualshock 3 controller currently still not support right?

**Comment by Masamune3210** [comment]
Unless something changed recently, yes, since the DS3 uses a very out of spec way of handling Bluetooth paring and connecting

**Comment by ndeadly** [comment]
This should fix the issue without the need for the flag file.
[MissionControl-0.6.3-moga-settsi-22a4074.zip](https://github.com/ndeadly/MissionControl/files/7704760/MissionControl-0.6.3-moga-settsi-22a4074.zip)

>  btw, the ps3 dualshock 3 controller currently still not support right?

Correct. I did some work on it a couple of months ago after hearing from the developer of Bloopair for WiiU that he was able to make it work (though he doesn't face some of the restrictions we do on the Switch), so I've updated it's status from "probably not feasible" to "likely possible with enough work". I've hit a bit of a roadblock on it for now and have been working on other things. Definitely requires some extensive debugging and patches to the `bluetooth` service. This will be a bit easier now with Atmosphere's gdbstub released, but I haven't had the time lately to resume work on it.
